virttest/utils_misc.py

```diff
@@ -2105,19 +2105,27 @@ def get_cpu_vendor(cpu_info="", verbose=True):
     return vendor
 
 
-def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
+def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm", remove_alias=False):
     """
-    Get the machine type the host support,return a list of machine type
+    Get the machine type the host support,return a tuple (s, c, v),
+    s, c, v are list, such as:
+    'pc', 'RHEL 7.4.0 PC (i440FX + PIIX, 1996)', '(alias of pc-i440fx-rhel7.4.0)'
     """
     o = process.system_output("%s -M ?" % qemu_binary).splitlines()
     s = []
     c = []
+    v = []
+    split_pattern = re.compile(r'^(\S+)\s+(.*?)(?: (\((?:alias|default).*))?$')
     for item in o[1:]:
-        machine_list = re.split(r"\s+", item, 1)
-        if machine_list[0] != "none":
-            s.append(machine_list[0])
-            c.append(machine_list[1])
-    return (s, c)
+        if remove_alias and "alias" in item:
+            continue
+        if "none" in item:
+            continue
+        machine_list = split_pattern.search(item).groups()
+        s.append(machine_list[0])
+        c.append(machine_list[1])
+        v.append(machine_list[2])
+    return (s, c, v)
```

Comment:
if `v` is not used, why return it?

Reply:
Currently, `v`  was not used in the code that call this function. But keep the full output here is more reasonable. It is optional to use it or not when call this function.